src/components/PeopleManagement/tests/CreateGroupModal.test.jsx

```diff
@@ -0,0 +1,132 @@
+    enterpriseSlug,
+    enableLearnerPortal: true,
+    enterpriseFeatures: {
+      topDownAssignmentRealTimeLcm: true,
+      enterpriseGroupsV1: true,
+    },
+  },
+};
+
+const defaultProps = {
+  isModalOpen: true,
+  closeModal: jest.fn(),
+  enterpriseUUID: 'test-uuid',
+};
+
+const CreateGroupModalWrapper = ({
+  initialState = initialStoreState,
+}) => {
+  const store = getMockStore({ ...initialState });
+  return (
+    <IntlProvider locale="en">
+      <Provider store={store}>
+        <QueryClientProvider client={queryClient()}>
+          <CreateGroupModal {...defaultProps} />
+        </QueryClientProvider>
+      </Provider>
+    </IntlProvider>
+  );
+};
+
+describe('<InviteMemberModal />', () => {
+  it('Modal renders as expected', async () => {
+    render(<CreateGroupModalWrapper />);
+    expect(screen.getByText('Create a custom group of members')).toBeInTheDocument();
+    expect(screen.getByText('Name your group')).toBeInTheDocument();
+    expect(screen.getByText('Select group members')).toBeInTheDocument();
+    expect(screen.getByText('Upload a CSV or select members from the table below.')).toBeInTheDocument();
+    expect(screen.getByText('You haven\'t uploaded any members yet.')).toBeInTheDocument();
+    expect(screen.getByText('Upload a CSV file or select members to get started.')).toBeInTheDocument();
+    expect(screen.getByText('Create')).toBeInTheDocument();
+    expect(screen.getByText('Cancel')).toBeInTheDocument();
+  });
+  it('creates groups and assigns learners', async () => {
+    const mockCreateGroup = jest.spyOn(LmsApiService, 'createEnterpriseGroup');
+    const mockInvite = jest.spyOn(LmsApiService, 'inviteEnterpriseLearnersToGroup');
+
+    const mockGroupData = { uuid: 'test-uuid' };
+    LmsApiService.createEnterpriseGroup.mockResolvedValue({ status: 201, data: mockGroupData });
+
+    const mockInviteData = { records_processed: 1, new_learners: 1, existing_learners: 0 };
+    LmsApiService.inviteEnterpriseLearnersToGroup.mockResolvedValue(mockInviteData);
+
+    render(<CreateGroupModalWrapper />);
+    expect(screen.getByText('You haven\'t uploaded any members yet.')).toBeInTheDocument();
+    expect(screen.getByText('Upload a CSV file or select members to get started.')).toBeInTheDocument();
+    const fakeFile = new File(['tomhaverford@pawnee.org'], 'emails.csv', { type: 'text/csv' });
+    const dropzone = screen.getByText('Drag and drop your file here or click to upload.');
+    Object.defineProperty(dropzone, 'files', {
+      value: [fakeFile],
+    });
+    fireEvent.drop(dropzone);
+
+    await waitFor(() => {
+      expect(screen.getByText('emails.csv')).toBeInTheDocument();
+      expect(screen.getByText('Summary (1)')).toBeInTheDocument();
+      expect(screen.getByText('Total members to invite')).toBeInTheDocument();
+      expect(screen.getByText('tomhaverford@pawnee.org')).toBeInTheDocument();
+      const formFeedbackText = 'Maximum invite at a time: 1000';
```

Comment:
The feedback in Figma is "Maximum **members** at a time: 1,000 emails" to avoid using the word invite